src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -180,19 +180,33 @@ def _try_handle_high_volume_task_trigger(ts: datetime):
     reference_datetime = ts.replace(second=0, microsecond=0)
     reference_ts = int(reference_datetime.timestamp())
 
+    existing_last_ts = redis_client.get(HIGH_VOLUME_LAST_TRIGGER_TS_KEY)
```

Comment:
I don't particularly LOVE this variable name, any thoughts?

Reply:
`precheck_last_ts`? Idk if that's much better

Reply:
Feels better 👍